src/winbind.c

```diff
@@ -245,6 +247,15 @@ uint32_t winbind_srv_auth(char *user, char *domain,
 
     memcpy(ntlmv2_key->data, wbc_info->user_session_key, ntlmv2_key->length);
 
+    // num_sids is INOUT: IN - maximum number of sids to be returned
```

Comment:
`/* */`-style comments please.

Reply:
Yes, I kept this style in mind but missed it here...